doc/howtos/vulnerability-management.rst

```diff
@@ -0,0 +1,145 @@
+Ostro OS vulnerability handling
+###############################
+
+Introduction
+============
+
+Software vulnerabilities are found continuously even in mature pieces
+of software. This means that one of the key objectives of a Linux
+distribution is pushing fixes to vulnerabilities down to the end users.
+Ostro OS images will not be used as-is by end users, but Ostro OS
+will be the base software distribution for many products the Ostro OS
+customers are developing. These products must be kept secure for their
+expected lifetime. The published Ostro OS images are for development
+purposes only and should not be used in real products.
+
+Finding vulnerabilities
+=======================
+
+The Ostro Project security team must find out about vulnerabilities in
+Ostro OS. These can be new ones or introduced when Ostro OS content is
+changed. The `cve-check-tool <https://github.com/ikeydoherty/cve-check-
+tool>`_ is run during Ostro OS builds. The tool creates a file that
+maps current Ostro OS recipes against known vulnerabilities for the
+particular software versions. This file provides information about open
+CVE issues affecting components in Ostro OS, but has some limitations
+because not all CVE issues are exactly mapped to packages. For those
+components, such as Linux kernel, CVEs and vulnerability reports have
+to be investigated separately. Also, cve-check-tool doesn't report
+vulnerabilities that are not found in the vulnerability database.
+
+When Ostro OS -specific vulnerabilities are found that don't have a CVE
+entry, the person finding them can notify the Ostro OS Security Team by
+reporting the vulnerability using the secure contact email address and
+public key for communication encryption.
+
```

Comment:
Link to the page with the email and public key info here.  I didn't see it at a glance from https://ostroproject.org/#get-involved so it may need to be created and linked appropriate there alongside the bug reporting link.